partner_label/tests/test_partner_label.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
```

Comment:
from odoo

Reply:
And this one

Reply:
thanks, for exactly this repo my checkers were botched and didn't actually check anything...